### DISCUSSION TITLE
WDIO v8 - Element not found second screen :: Native apps (Android/iOS)

### DISCUSSION THREAD

**Discussion by aftab-mannan** [webdriverio/webdriverio#9540] [Q&A]
Hi All,

I am facing 'Element not found error' on the second screen which also happens to be a bottom sheet/layover screen.
I can click on a button on the first screen to navigate to the second screen. 

I have verified the locators and it works fine in Java/Appium. Please suggest

**Comment by christian-bromann** [comment]
@aftab-mannan please understand that with these limited amount of information it is impossible to say what is going on. I recommend debug the issue using e.g. the `debug` command.
